include/shell/shell.h

```diff
@@ -837,6 +837,18 @@ void shell_help(const struct shell *shell);
  */
 int shell_execute_cmd(const struct shell *shell, const char *cmd);
 
+/** @brief Set root command for all shell instances.
+ *
+ * It is an equivalent of calling select command with one of the root commands
```

Comment:
In my opinion, a description of the commit message a bit better highlights the differences between these commands.

> It allows setting from the code the root command. It is an equivalent of calling 'select ' except it sets command for all shell instances.